test/_import/jx_import.go

```diff
@@ -53,9 +71,8 @@ var _ = Describe("Import Application", func() {
 					Expect(destDir + "/.git").ToNot(BeADirectory())
 				})
 
-				By("updating the pom.xml to have the correct application name", func() {
-					err := utils.ReplaceElement(filepath.Join(destDir, "pom.xml"), "artifactId", T.ApplicationName, 1)
-					Expect(err).NotTo(HaveOccurred())
+				By("updating the pom.xml (if exists) to have the correct application name", func() {
+					_ = utils.ReplaceElement(filepath.Join(destDir, "pom.xml"), "artifactId", T.ApplicationName, 1)
```

Comment:
Should we be checking the error here?

Reply:
Good point! But we actually don't, because only the maven quickstart requires the pom file to be updated so that it doesn't fail when pushing the jar to nexus. The other quickstarts don't have this problem, or even have a `pom.xml` file. Since the file is not there, this fails for all the other quickstarts, but it's fine! We could add some conditionals here and there, but I thought it was cleaner this way. Is there a better way to go about it?

Reply:
I think it's been removed because the import tests are now testing more than just java spring quick starts, so the pom.xml may not exist for some tests. But you're right, maybe we should be checking the type of error and ignoring if its file not found or only run this step if it is a spring quick start.

Reply:
I've added some logic to fail only if we get an error which is not the error that occurs when the file doesn't exist.

Reply:
looks good